Gemfile

```diff
@@ -1,2 +1,12 @@
 source 'https://rubygems.org'
+
+gem 'jeweler'
+gem 'pry'
+gem 'rake'
+gem 'rdoc'
+gem 'rspec'
```

Comment:
What do you think about putting major versions here (eg `gem 'rspec', '~> 3.0')`? Not necessary?

Reply:
I my opinion, absolutely not necessary. In [all my gems](http://github.dannyben.com/ruby/) I am using the same approach. We want devs to always use the latest. This way, you make it a habbit to occasionally do `bundle update` and `bundle outdated`, to ensure everything is new.

The only reason to occasionally lock versions, is when something is broken and we do not have time to fix it.

I am no expert on these matters - but this is how I roll.

Reply:
Got it, that sounds good to me.